docs/whatsnew/1.3.rst

```diff
@@ -65,6 +65,14 @@ Coordinate transformations can use JPL Ephemerides
 Earth locations can now be obtained by address
 ----------------------------------------------
 
+With the new :meth:`~astropy.coordinates.EarthLocation.of_address` classmethod,
+:class:`~astropy.coordinates.EarthLocation` objects can now be easily created
+using a string address, for example, "350 5th Ave, New York, NY 10118". This
+works by querying the Google Maps API to retrieve the latitude, longitude, and
+(optional) height of the specified location. This can be useful for quickly
```

Comment:
Could you actually show the example you mention in 2 lines of code? (it's nice to have some small code examples in the 'what's new'